public class Main {
    public static void main(String[] args) throws Exception {
        RecFile.recFile();
        RecFile_3.recFile();
        System.out.println();
        System.out.println("Задача № 1");
        System.out.println();
        Task_1.task_1();
        System.out.println();
        System.out.println("Задача № 2");
        System.out.println();
        Task_2.task_2();
        System.out.println();
        System.out.println();
        System.out.println("Задача № 3");
        System.out.println();
        Task_3.task_3();
        System.out.println();
        System.out.println();
        System.out.println("Задача № 4");
        System.out.println();
        Task_4.task_4();
    }
}


